### PR TITLE
Story css

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me

File: frontend/src/GlobalStyle.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ justify-content: flex-start;
   --font-family: 'Francois One', sans-serif;
   --text-shadow: 2px 2px 6px var(--black);
   --box-shadow: 2px 2px 5px var(--black);
+
```
Comment (LoicBrassart) [inline frontend/src/GlobalStyle.jsx:18]: (Why go the extra mile to edit a file for **just** a useless empty line ?)

**Review by LoicBrassart — APPROVED** [review]
SnE (except for a useless empty line)

**Pull request merged**